### PR TITLE
Create 8.x-2.x

### PR DESCRIPTION
**Pull request by recrit**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -8,41 +8,46 @@
...
     "jcalderonzumba/mink-phantomjs-driver": "~0.3.1",
     "mikey179/vfsStream": "~1.2",
+    "se/selenium-server-standalone": "^2.53",
     "phpunit/phpunit": "~4.8",
     "drush/drush": "^8",
     "drupal/console": "~1.0.0",
     "phpmd/phpmd": "~2.1",
-    "drupal/admin_toolbar": "~8.1",
+    "drupal/admin_toolbar": "^1.18",
     "drupal/coder": "^8.2",
```
Comment (mike-potter) [inline composer.json]: Doesn't drupal/coder also need a new version for the d.o packagist? Like ^2.0 or something.

File: composer.json [hunk 2]
```diff
@@ -8,41 +8,46 @@
...
-    "drupal/paragraphs": "dev-8.x-1.x",
-    "drupal/restui": "~8.1",
-    "drupal/search_api": "~8.1",
-    "drupal/search_api_solr": "dev-8.x-1.x",
-    "drupal/search_api_page": "~8.1",
-    "drupal/facets": "dev-8.x-1.x",
-    "drupal/components": "~8.1"
+    "cweagans/composer-patches": "~1.6",
+    "acquia/lightning": "~2.0.0",
+    "drupal/core": "~8.2.0",
```
Comment (mike-potter) [inline composer.json]: You shouldn't need to specify "drupal/core" here. Composer *should* take the dependency given in the Lightning distro itself.

File: composer.json (hunk 1 above)
Comment (recrit) [reply composer.json]: checked and i haven't had any errors when building. The composer.lock is showing 8.2.10 from https://github.com/klausi/coder.git.
I'll update to "^2.0"

File: composer.json [hunk 3]
```diff
@@ -8,41 +8,46 @@
...
+    "drupal/diff": "^1.0",
+    "drupal/field_group": "^1.0",
+    "drupal/paragraphs": "1.x-dev",
+    "drupal/restui": "^1.0",
+    "drupal/search_api": "^1.0",
```
Comment (mike-potter) [inline composer.json]: Latest version of Lightning includes search_api 1.0.

**Pull request merged**